Framework/DataHandling/inc/MantidDataHandling/DataBlock.h

```diff
@@ -7,11 +7,14 @@
 #pragma once
 
 #include "MantidDataHandling/DllConfig.h"
+#include "MantidGeometry/IDTypes.h"
 #include "MantidNexus/NexusClasses.h"
 
 namespace Mantid {
 namespace DataHandling {
 
+using spectrumPair = std::pair<specnum_t, specnum_t>;
```

Comment:
Sorry to be picky but can composite type aliases follow naming conventions of classes, i.e.  `SpectrumPair`. It's aliasing a non-basic type so it would be good to be consistent with other classes. Also if this were replaced by an actual class or struct its name would not need to change.

Reply:
Good point 👍. It had slipped my mind on how to name these type aliases, `SpectrumPair` is definitely more appropriate here.